Dashboard tile

JulianaChiabai opened the pull request.

JulianaChiabai committed:
Dashboard tile

JulianaChiabai reviewed (commented):
.

JulianaChiabai committed:
Dashboard tile

JulianaChiabai committed:
- Wijzigingen in 'composer.json'
- UI aangepast
- Toon 4 gebruikers

JulianaChiabai committed:
- design aangepast

tomcoonen commented on src/GitLabTileServiceProvider.php:
Would be good to make this tile more specific, like `GitLabUserCountsTitle` so it reflects what is does.

tomcoonen commented on tests/ExampleTest.php:
Test namespace is wrong.

tomcoonen committed:
Update FetchDataFromApiCommand.php

Removed usernames

tomcoonen commented on src/FetchDataFromApiCommand.php:
API endpoint should be configurable, and we should show initials on the tile instead of full usernames so it's more in line with the rest.

tomcoonen commented on CHANGELOG.md:
Replace `:package_name` please.

JulianaChiabai committed:
- replaced package_name
- name space fixed
- updated .env to include GITLAB_API_URL
- hid usernames in configuration
- renamed files

JulianaChiabai committed:
Merge branch 'starter' of github.com:creacoon/laravel-dashboard-gitlab-user-counts-tile into starter

# Conflicts:
#	src/FetchDataFromGitLabUserCountsCommand.php

tomcoonen commented on src/FetchDataFromGitLabUserCountsCommand.php:
```diff
@@ -0,0 +1,58 @@
+    public function handle()
+    {
+        $gitlabConfig = config('dashboard.tiles.gitlab');
+        $specificUsers = $gitlabConfig['specific_users'];
```
Please use the configs directly, this helps them to be found, and you can easily click through in the IDE.
So e.g. `config('dashboard.tiles.gitlab.specific_users')`.

JulianaChiabai committed:
- configs used directly

JulianaChiabai committed:
- configs used directly

JulianaChiabai committed:
- composer.json

JulianaChiabai committed:
- composer.json

JulianaChiabai committed:
- composer.json

JulianaChiabai committed:
- README.md

JulianaChiabai committed:
- README.md

tomcoonen committed:
Show user initials. Get user counts for all users. Readme improvements.
